examples/aks-spring-music/README.md

```diff
@@ -37,7 +37,7 @@ The bundle will use the service principal created above to interact with Azure.
     ```
 
 * Update params for your deployment
-    * change the `invocationImage` Docker repo to match your Docker Hub account (line 4)
+    * change the `tag` Docker repo to match your Docker Hub account (line 5)
```

Comment:
nit/suggestion: leave off the line # as it's easy to drift without us noticing

Reply:
Good call; will remove.